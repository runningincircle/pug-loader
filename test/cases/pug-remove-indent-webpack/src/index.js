const html = require('./indent.pug');

console.log(html);