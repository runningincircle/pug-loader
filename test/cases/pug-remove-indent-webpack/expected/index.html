<h1>Hello World!</h1><p>text</p><div><p>text</p><p>text</p></div><div>block</div>